galaxy/api/serializers/survey.py

```diff
@@ -36,14 +36,17 @@
             'ease_of_use',
             'does_what_it_says',
             'works_as_is',
-            'used_in_production'
+            'used_in_production',
+            'content_id'
         )
 
 
 class CollectionSurveySerializer(serializers.BaseSerializer):
+    content_id = drf_serializers.SerializerMethodField()
```

Comment:
You can use `source=` attribute instead of `SerializerMethodField` to access nested object attributes.